CHANGELOG.md

```diff
@@ -26,9 +26,7 @@
 - PR #2340: Import ARIMA in the root init file and fix the `test_fit_function` test
 - PR #2408: Install meta packages for dependencies
 - PR #2417: Move doc customization scripts to Jenkins
-- PR #2411 Refactor Mixin classes and use in classifier/regressor estimators
-- PR #2099: Raise an error or convert dtype when float64 data is used with dask RF
-- PR #2403 Support for input and output type consistency in logistic regression predict_proba
+- PR #2427: Moving MNMG decomposition to cuml
```

Comment:
Weird changelog diff? Maybe need to update?

Reply:
Ohh i dont see it, this might have been that github did not refresh the page right